bluebrain/deployment/environments/externals.yaml

```diff
@@ -59,7 +59,7 @@ spack:
       # The general cuda should be set via the generic packages.yaml!
       variants: +cuda cuda_arch=70 +omp_tsan +python +link_dylib
     mvapich2:
-      variants: fabrics=mrail process_managers=slurm file_systems=gpfs
+      variants: fabrics=mrail process_managers=slurm file_systems=gpfs,ime
```

Comment:
the package says following 😒 : 
![image](https://user-images.githubusercontent.com/666852/180453259-b88c29c8-307f-4785-8399-213b3a6e5951.png)